manifests/generated/kv-resource.yaml

```diff
@@ -424,6 +424,21 @@ spec:
                     description: PullPolicy describes a policy for if/when to pull
                       a container image
                     type: string
+                  instancetype:
+                    description: Instancetype configuration
+                    nullable: true
+                    properties:
+                      referencePolicy:
+                        description: |-
+                          ReferencePolicy defines how an instance type or preference should be referenced by the VM after submission, supported values are:
+                          reference (default) - Where a copy of the original object is stashed in a ControllerRevision and referenced by the VM.
```

Comment:
This part of the core API so we have to be careful changing it. We should consider if we want to abolish ControllerRevisions altogether? (also if time permits)

Reply:
Agreed, I did try to add this to the `instancetype.kubevirt.io` group but that code actually imports this core group and thus can't be used here without causing an import loop.

The removal of ControllerRevisions is a topic in the design proposal but I don't think it provides us anything and is such a massive behavioural change that there's zero chance of it landing before FF.

At least this is self contained, non-default, can be deprecated and removed in the future if not used anymore etc.